clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -36,7 +39,7 @@
     implements ServiceCreationConfiguration<ClusteringService>,
     CacheManagerConfiguration<PersistentCacheManager> {
 
-  private static final String CLUSTER_SCHEME = "terracotta";
+  private static final Collection<String> CLUSTER_SCHEMES = new HashSet<String>(Arrays.asList("terracotta", "passthrough"));
```

Comment:
I would like to see this relaxed to allow any URI here.  Imho it's up to Voltron to make a statement on what a legal identifier is or isn't.  To that end I'd even contemplate advocating only stripping the last element of the path, and using the remainder as part of the cluster URI.